presto-orc/src/main/java/com/facebook/presto/orc/metadata/statistics/BinaryStatisticsBuilder.java

```diff
@@ -25,6 +25,8 @@ public class BinaryStatisticsBuilder
         implements SliceColumnStatisticsBuilder
 {
     private long nonNullValueCount;
+    private long size;
```

Comment:
If I remember correctly we don't use a base class for stats builders to save some memory used by inheritance.